jobs/delorean/2.0/ews/3scale/discovery.yaml

```diff
@@ -0,0 +1,34 @@
+---
+- job:
+    name: delorean-2-ews/3scale/3scale_brew_build_created
+    display-name: 3Scale Brew Build Created
+    description: |
+      <h2><a href="https://github.com/integr8ly/ci-cd/tree/master/jobs/delorean/2.0/ews/3scale">Click here to see the job definition on GitHub!</a></h2>
+
+      <h3>This job gets triggered by Brew Builds on the UMB</h3>
```

Comment:
Might be an idea to put the brewweb link fort he package in here https://brewweb.engineering.redhat.com/brew/packageinfo?packageID=74069